primefaces/src/main/resources/META-INF/resources/primefaces/locales/locale-es.js

```diff
@@ -71,11 +71,11 @@ if (window.PrimeFaces) {
       pageLabel: "Página {page}",
       firstPageLabel: "Primera Página",
       lastPageLabel: "Última Página",
-      nextPageLabel: "Siguiente Página",
+      nextPageLabel: "Página Siguiente",
```

Comment:
I think "Siguiente Página"was ok.

Reply:
In Spanish/Castellan, you need to folow some semantic rules.

If you use

`nextPageLabel: "Página Siguiente",` 

then you need to change

`previousPageLabel: "Página Anterior",`

to

`previousPageLabel: "Anterior Página",`

because they have the same semantic context.

It's more correct say

"Página Siguiente"

example:

- Veamos la página siguiente (rigth)

- Veamos la siguiente página (rigth, understable but not ussually used)

Reply:
In my opinion is more usually use ''Siguiente página'' than ''Página siguiente''

Reply:
Where are you from?